.pre-commit-config.yaml

```diff
@@ -45,9 +45,15 @@ repos:
         exclude: ^src/templates/python$
 
   - repo: https://github.com/codespell-project/codespell
-    rev: "v2.2.6"
+    rev: "v2.3.0"
     hooks:
       - id: codespell
+        args:
+          [
+            "--ignore-words-list",
+            "ags,aray,asend,ba,crate,falsy,feld,inflight,lits,nd,slowy,te,oint,conveniant,atmost",
+          ]
+        exclude: ^(benchmark/benchmarks/pystone_benchmarks/pystone\.py|src/js/package-lock\.json)$
```

Comment:
We added [codespell_ignore_words.txt](https://github.com/pyodide/pyodide/blob/main/tools/codespell_ignore_words.txt) file to manage this list (https://github.com/pyodide/pyodide/pull/4938). So this change needs to be reverted.

Reply:
ah, sorry I likely messed up the rebase.